clang/lib/Basic/Targets/OSTargets.h

```diff
@@ -231,6 +231,9 @@ class LLVM_LIBRARY_VISIBILITY FreeBSDTargetInfo : public OSTargetInfo<Target> {
     case llvm::Triple::riscv32:
     case llvm::Triple::riscv64:
       break;
+    case llvm::Triple::loongarch32:
```

Comment:
I'm of two minds about adding loongarch32 triples. It's harmless and parallels riscv, but the chances of a new 32-bit port being importing into the FreeBSD are close to zero.

Reply:
@brooksdavis Thanks!
after have reading [Future of 32-bit platform support in FreeBSD](https://lists.freebsd.org/archives/freebsd-announce/2024-February/000117.html) , I think It's reasonable to exclude 32 bit from LoongArch support. 
I also hope to hear some comments/suggestions from @emaste

Reply:
@brooksdavis 32-bit RISC-V support was removed for FreeBSD. I just noticed this addition and think it should be removed.